ducktape/tests/reporter.py

```diff
@@ -187,17 +187,31 @@ def test_results_dir(self, result):
         test_results_dir = os.path.abspath(result.results_dir)
         return test_results_dir[len(base_dir):]  # truncate the "absolute" portion
 
+    def append_result(self, result_string, result):
```

Comment:
Seems this ended up unused?

Reply:
Ah, yeah, sorry about that. Had it initially, but felt like overkill in the end.